src/test/java/bio/terra/service/dataset/flight/ingest/IngestUtilsTest.java

```diff
@@ -66,4 +66,12 @@ public void testNoDoubleDash() {
     IngestUtils.validateBlobAzureBlobFileURL(
         "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata/test/azure-simple--dataset-ingest-request.csv");
   }
+
+  @Test
+  public void testValidate() {
+    IngestUtils.validateBlobAzureBlobFileURL(
+        //
+        // "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata/test/azure-simple-dataset-ingest-request.csv");
+        "https://pfizerhackrawdata2.blob.core.windows.net/pfizerstorageexplorermig/flatiron/fmi-specimens.csv");
```

Comment:
Did you mean to use a blob url from the hackathon in this test?

Reply:
gah, shoot, sorry, no I didn't.  I was testing a case where we don't accept valid URL (such as this one).  Separate ticket coming for that one